lib/src/components/layout/accordion.dart

```diff
@@ -63,20 +203,35 @@ class _AccordionItemState extends State<AccordionItem>
   late AnimationController _controller;
   late CurvedAnimation _easeInAnimation;
 
+  bool _initialized = false;
+
   @override
-  void initState() {
-    super.initState();
-    // _expanded = widget.expanded;
+  void didChangeDependencies() {
+    super.didChangeDependencies();
+
+    _AccordionState newAccordion = Data.of<_AccordionState>(context);
+    if (newAccordion != accordion) {
+      accordion?._expanded.removeListener(_onExpandedChanged);
+      newAccordion._expanded.addListener(_onExpandedChanged);
+      accordion = newAccordion;
+    }
+
+    if (_initialized) return;
+
+    _initialized = true;
+
+    final theme = ComponentTheme.maybeOf<AccordionItemTheme>(context);
+
     _expanded.value = widget.expanded;
```

Comment:
This will cause the state to reset to the initial value whenever widget dependencies notify its update.

Reply:
only when we remove the init variable. however i think it would be smarter to move this back into initState again

Reply:
I don't think you can fetch the theme inside initState